$NetBSD: patch-CMakeLists.txt,v 1.2 2025/10/22 14:43:49 gdt Exp $

Cope with boost deprecations.

Sent upstream by email 20251021, because their bugtracker both
requires and prohibits registrations.  Filed upstream 20251022 on
their other bugtracker.

  https://osmocom.org/issues/6870

--- CMakeLists.txt.orig	2024-05-15 17:40:15.000000000 +0000
+++ CMakeLists.txt
@@ -119,7 +119,7 @@ endif()
 ########################################################################
 # Find boost
 ########################################################################
-find_package(Boost "1.65" REQUIRED chrono thread system)
+find_package(Boost "1.65" REQUIRED chrono thread)
 
 if(NOT Boost_FOUND)
     message(FATAL_ERROR "Boost required to compile osmosdr")
